src/lib.rs

```diff
@@ -509,27 +536,90 @@ impl<T> Webauthn<T> {
         Ok(data.authenticatorData.counter)
     }
 
+    /// Generate a challenge for an authenticate request for a user. You must supply the set of
+    /// credentials that exist for the user that *may* be used in this authentication request. If
+    /// an empty credential set is supplied, the authentication *will* fail.
+    ///
+    /// This challenge is supplied to
+    /// to the client javascript function `navigator.credentials.get()`.
+    ///
+    /// You must persist the AuthenticationState that is returned. You should associate this by
+    /// UserId. The AuthenticationState is required for the authenticate_credential function to
+    /// operate correctly.
+    ///
+    /// At this time we deviate from the standard and base64 some fields, but we are
+    /// investigating how to avoid this (https://github.com/Firstyear/webauthn-rs/issues/5)
+    pub fn generate_challenge_authenticate(
+        &mut self,
+        _username: &UserId,
```

Comment:
It looks like this field is unnecessary?

Reply:
And removed :)